test/isArgument_test.js

```diff
@@ -0,0 +1,47 @@
+goog.require('helper');
+
+describe('The helper.isArguments_ function', function() {
```

Comment:
I think these tests cover all the most important cases, but for really simple but fundamental methods like this I like to have a very wide variety of test cases.  I'd suggest adding cases for:

- All falsey types, not just null.  So null, undefined, 0, false, and empty string.
- A few other simple types (e.g. positive number, non-empty string, true).
- A few additional complex types (e.g. a plain object, a constructed object).

It's also fine to make multiple assertions in a single test method, if they make sense together.  e.g. grouping all the falsey types into `it('Identifies falsey values as not an arugments object', ...` would be reasonable.

Reply:
Got it. Just out of curiosity, is there a reason why you say a positive number specifically in the simple type examples? I read that javascript numbers are all floating-point so I was curious if there was any difference that I wasn't aware of.